RecoLocalCalo/HGCalRecProducers/plugins/HGCalUncalibRecHitProducer.cc

```diff
@@ -12,7 +12,7 @@
 HGCalUncalibRecHitProducer::HGCalUncalibRecHitProducer(const edm::ParameterSet& ps) :
   eeDigiCollection_( consumes<HGCEEDigiCollection>( ps.getParameter<edm::InputTag>("HGCEEdigiCollection") ) ),
   hefDigiCollection_( consumes<HGCHEDigiCollection>( ps.getParameter<edm::InputTag>("HGCHEFdigiCollection") ) ),
-  hebDigiCollection_( consumes<HGCHEDigiCollection>( ps.getParameter<edm::InputTag>("HGCHEBdigiCollection") ) ),
+  hebDigiCollection_( consumes<HGCBHDigiCollection>( ps.getParameter<edm::InputTag>("HGCHEBdigiCollection") ) ),
```

Comment:
should the input tag be renamed to keep the naming convention?
Now you'd have \*HEB\*  named parameter specify \*BH\* collection

If I'm not mistaken, the design documentation refers to these as FH and BH sections.
It would be much less confusing to keep this in C++ and PSet names.